root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4984,6 +4989,55 @@ function doMatchSubtest(
   previousMatchTests.push(entityType + '+' + url);
 }
 
+function doOptionsSubtest(
+  st,
+  entityType,
+  url,
+  label,
+  expectedRelationshipOptions,
+) {
+  const checker = new Checker(cleanURL(url), entityType);
+  const possibleOptions = checker.getPossibleTypes();
+  let actualRelationshipOptions = possibleOptions || undefined;
+  if (expectedRelationshipOptions === undefined) {
+    actualRelationshipOptions = undefined;
+  }
+
+  // The options can be just one relationship type, or arrays of 2+ types
+  const relationshipOptions = possibleOptions.map(optionSet => {
```

Comment:
I would suggest to consistenly use the term “combination” that is already used for this matter: `relationshipTypeCombinations`.